### PR TITLE
Enable some tests for Inference Engine 2019R1

### PR DESCRIPTION
**Pull request by dkurt**
### This pullrequest changes

**Merge with extra**: https://github.com/opencv/opencv_extra/pull/592

<cut/>

```
force_builders=Custom,Linux AVX2
buildworker:Custom=linux-2
docker_image:Custom=ubuntu-openvino-2019r1:16.04
test_opencl:Custom=ON
test_bigdata:Custom=1
test_filter:Custom=*
test_modules:Custom=dnn

ci_branch:Linux AVX2=no-checks
docker_image:Linux AVX2=ubuntu-openvino-2018r5:16.04
test_opencl:Linux AVX2=OFF
buildworker:Linux AVX2=linux-1
build_parallel_tests:Linux AVX2=1
test_bigdata:Linux AVX2=1
test_module:Linux AVX2=dnn,python2,python3,java
tests_filter:Linux AVX2=*

buildworker:Mac OpenCL=macosx-1
build_image:Mac OpenCL=openvino-2019r1
```

### CODE REVIEW

**Comment by dkurt** [comment]
There is something wrong with Myriad plugin. 2 sec initialization was observed here https://github.com/opencv/opencv/pull/13692 (due plugin crash).
```
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/358, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 1x0, 2x2, true, OCV/OCL)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/358 (2 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/359, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 1x0, 2x2, true, OCV/OCL_FP16)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/359 (2 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/360, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, false, DLIE/CPU)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/360 (3 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/361, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, false, DLIE/MYRIAD)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/361 (2014 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/362, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, false, DLIE/OCL)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/362 (265 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/363, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, false, DLIE/OCL_FP16)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/363 (251 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/364, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, false, OCV/OCL)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/364 (1 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/365, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, false, OCV/OCL_FP16)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/365 (0 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/366, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, true, DLIE/CPU)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/366 (3 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/367, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, true, DLIE/MYRIAD)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/367 (2013 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/368, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, true, DLIE/OCL)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/368 (254 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/369, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, true, DLIE/OCL_FP16)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/369 (252 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/370, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, true, OCV/OCL)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/370 (2 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/371, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 1x1, true, OCV/OCL_FP16)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/371 (1 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/372, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 2x2, false, DLIE/CPU)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/372 (4 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/373, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 2x2, false, DLIE/MYRIAD)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/373 (2013 ms)
[ RUN      ] Layer_Test_Halide/Convolution.Accuracy/374, where GetParam() = ([6, 9, 1], 5x6, 1x3, 2x2, 0x1, 2x2, false, DLIE/OCL)
[       OK ] Layer_Test_Halide/Convolution.Accuracy/374 (238 ms)
```

File: modules/dnn/test/test_precomp.hpp [hunk 1]
```diff
@@ -47,6 +47,6 @@
 #include "opencv2/core/ocl.hpp"
 
 #include "opencv2/dnn.hpp"
-#include "test_common.hpp"
+#include "../test/test_common.hpp"
```
Comment (alalek) [inline modules/dnn/test/test_precomp.hpp]: Does this change required?

Comment (dkurt) [reply modules/dnn/test/test_precomp.hpp]: I have
```
fatal error: test_common.hpp: No such file or directory
```
with the following configuration:

```
cmake -DCMAKE_INSTALL_PREFIX=/usr/local \
  -DCMAKE_BUILD_TYPE=Release \
  -DWITH_OPENCL=ON \
  -DWITH_LAPACK=OFF \
  -DWITH_JASPER=OFF \
  -DBUILD_DOCS=OFF \
  -DBUILD_TESTS=ON \
  -DWITH_OPENMP=OFF \
  -DBUILD_PERF_TESTS=ON \
  -DWITH_OPENEXR=OFF \
  -DWITH_WEBP=OFF \
  -DWITH_TIFF=OFF \
  -DWITH_INF_ENGINE=ON \
  -DWITH_VULKAN=OFF \
  -DENABLE_CXX11=ON \
  -DBUILD_EXAMPLES=OFF \
  -DINF_ENGINE_RELEASE=2019010000 \
  -DBUILD_LIST=ts,dnn,python2 ..
```

note: without IE it's ok.

Comment (dkurt) [reply modules/dnn/test/test_precomp.hpp]: removed it

**Pull request merged**